records/generator.go

```diff
@@ -22,16 +22,15 @@ import (
 // Will likely become map[string][]discoveryinfo
 type rrs map[string][]string
 
-// Mesos-DNS state
-// Refactor when discovery id is available
+// RecordGenerator contains DNS records and methods to access and manipulate
+// them. TODO: Refactor when discovery id is available.
 type RecordGenerator struct {
 	As       rrs
 	SRVs     rrs
 	SlaveIPs map[string]string
 }
 
-// The following types help parse state.json
-// Resources holds our SRV ports
+// Resources holds SRV ports.
```

Comment:
In fact this is not about SRV ports in the DNS sense. It's just for the state.json structures from Mesos master.